tutorials/embedded/step-7.html.haml

```diff
@@ -0,0 +1,208 @@
+                  %pre
+                    %code.bash
+                      = preserve do
+                        :escaped        
+                          Jan 30, 2015 3:22:08 PM org.infinispan.remoting.transport.jgroups.JGroupsTransport start
+                          INFO: ISPN000078: Starting JGroups channel WeatherApp
+                          Jan 30, 2015 3:22:08 PM org.infinispan.remoting.transport.jgroups.JGroupsTransport buildChannel
+                          INFO: ISPN000088: Unable to use any JGroups configuration mechanisms provided in properties {}. Using default JGroups configuration!
+                          Jan 30, 2015 3:22:24 PM org.infinispan.remoting.transport.jgroups.JGroupsTransport viewAccepted
+                          INFO: ISPN000094: Received new cluster view for channel WeatherApp: [hoth-1712|0] (1) [hoth-1712]
+                          ---- View changed: [hoth-1712] ----
+                          Jan 30, 2015 3:22:24 PM org.infinispan.remoting.transport.jgroups.JGroupsTransport startJGroupsChannelIfNeeded
+                          INFO: ISPN000079: Channel WeatherApp local address is hoth-1712, physical addresses are [10.36.7.250:60802]
+                          Jan 30, 2015 3:22:24 PM org.infinispan.factories.GlobalComponentRegistry start
+                          INFO: ISPN000128: Infinispan version: Infinispan 'Hoptimus Prime' 7.1.0.Final
+                          ---- Waiting for cluster to form ----
+                          Jan 30, 2015 4:10:41 PM org.infinispan.remoting.transport.jgroups.JGroupsTransport viewAccepted
+                          INFO: ISPN000094: Received new cluster view for channel WeatherApp: [hoth-1712|1] (2) [hoth-1712, hoth-9822]
+                          ---- View changed: [hoth-1712, hoth-9822] ----
+                          ---- Fetching weather information ----
+                          Jan 30, 2015 4:10:41 PM org.infinispan.topology.ClusterTopologyManagerImpl broadcastRebalanceStart
+                          INFO: ISPN000310: Starting cluster-wide rebalance for cache ___defaultcache, topology CacheTopology{id=1, rebalanceId=1, currentCH=DefaultConsistentHash{ns = 60, owners = (1)[hoth-1712: 60+0]},                 pendingCH=DefaultConsistentHash{ns = 60, owners = (2)[hoth-1712: 30+30, hoth-9822: 30+30]}, unionCH=null, actualMembers=[hoth-1712, hoth-9822]}
+                          Jan 30, 2015 4:10:41 PM org.infinispan.topology.ClusterTopologyManagerImpl handleRebalanceCompleted
+                          INFO: ISPN000328: Finished local rebalance for cache ___defaultcache on node hoth-1712, topology id = 1
+                          Jan 30, 2015 4:10:41 PM org.infinispan.topology.ClusterTopologyManagerImpl handleRebalanceCompleted
+                          INFO: ISPN000328: Finished local rebalance for cache ___defaultcache on node hoth-9822, topology id = 1
+                          -- Entry for Rome, Italy modified by another node in the cluster
```

Comment:
Should "Entry for X modified by another node in the cluster" show in the coordinator log as well?